# origin: Debian (joshk)
# cset: n/a
# inclusion: not suitable for mainline
# description: fix spurious ampersand in new firmwareless tg3 code for 2.4.27
# revision date: 2004-09-03

--- a/drivers/net/tg3.c	2004-08-18 20:51:41.000000000 -0700
+++ b/drivers/net/tg3.c	2004-08-18 20:51:02.000000000 -0700
@@ -3911,7 +3911,7 @@
 	if (*info_ptr)
 		return 0;
 
-	err = request_firmware(&fw_entry, filename, pci_name(&tp->pdev));
+	err = request_firmware(&fw_entry, filename, pci_name(tp->pdev));
 	if (err) {
 		printk(KERN_WARNING PFX "tg3_request_firmware (%s): "
 		       "Couldn't get firmware \"%s\".\n",
